source/shared/core_sqlsrv.h

```diff
@@ -1099,24 +1103,28 @@ const char SERVER[] = "Server";
 
 enum SQLSRV_CONN_OPTIONS {
    
-   SQLSRV_CONN_OPTION_INVALID,
-   SQLSRV_CONN_OPTION_APP,
-   SQLSRV_CONN_OPTION_CHARACTERSET,
-   SQLSRV_CONN_OPTION_CONN_POOLING,
-   SQLSRV_CONN_OPTION_DATABASE,
-   SQLSRV_CONN_OPTION_ENCRYPT,
-   SQLSRV_CONN_OPTION_FAILOVER_PARTNER,
-   SQLSRV_CONN_OPTION_LOGIN_TIMEOUT,
-   SQLSRV_CONN_OPTION_MARS,
-   SQLSRV_CONN_OPTION_QUOTED_ID,
-   SQLSRV_CONN_OPTION_TRACE_FILE,
-   SQLSRV_CONN_OPTION_TRACE_ON,
-   SQLSRV_CONN_OPTION_TRANS_ISOLATION,
-   SQLSRV_CONN_OPTION_TRUST_SERVER_CERT,
-   SQLSRV_CONN_OPTION_WSID,
-   SQLSRV_CONN_OPTION_ATTACHDBFILENAME,
-   SQLSRV_CONN_OPTION_APPLICATION_INTENT,
-   SQLSRV_CONN_OPTION_MULTI_SUBNET_FAILOVER,
```

Comment:
Please fix indentation (somehow is has one less space thus it's saying you've modified these lines, while you'll only added the last 4 lines